modules/add-new-book.js

```diff
@@ -0,0 +1,25 @@
+import library from './library.js';
+import drawOnScreen from './drow-on-screen.js';
+
+function getInputValues() {
+  const bookEntries = {
+    title: null,
+    author: null,
+  };
+
+  const titleInput = document.getElementById('title');
+  const authorInput = document.getElementById('author');
+
+  bookEntries.title = titleInput.value;
+  bookEntries.author = authorInput.value;
+
+  library.add(bookEntries);
+
+  titleInput.value = '';
+  authorInput.value = '';
+
+  window.localStorage.setItem('books', JSON.stringify(library.booksArr));
+  drawOnScreen(bookEntries);
+}
```

Comment:
- [ ] This is very handy in getting the form input values. However, I will suggest you refactor it and utilize ES6 arrow function syntax for[ readability and more](https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/).

Example: 
```js
const getInputValues = () => {
 // add contents here
}
```